gateway-service-impl/src/main/java/org/hypertrace/gateway/service/baseline/BaselineServiceQueryParser.java

```diff
@@ -93,58 +95,44 @@ public QueryRequest getQueryRequest(
                 entityIdAttributes.stream()
                     .map(
                         entityIdAttribute ->
-                            QueryRequestUtil.createColumnValueFilter(
-                                entityIdAttribute, Operator.NEQ, QUERY_SERVICE_NULL))
-                    .map(Filter.Builder::build)
+                            createFilter(
+                                entityIdAttribute,
+                                Operator.NEQ,
+                                createStringNullLiteralExpression()))
                     .collect(Collectors.toList()));
 
     filterBuilder.addAllChildFilter(
         entityIdAttributes.stream()
             .map(
                 entityIdAttribute ->
-                    QueryRequestUtil.createValueInFilter(entityIdAttribute, entityIds))
-            .map(Filter.Builder::build)
+                    QueryRequestUtil.createFilter(
+                        entityIdAttribute,
+                        Operator.IN,
+                        createStringArrayLiteralExpression(new ArrayList<>(entityIds))))
```

Comment:
```suggestion
                        createStringArrayLiteralExpression(entityIds)))
```
Is there a reason to rewrap the list?

Reply:
Because of extra brackets used intellij suggested change, fixed it.